src/nvim/os/env.c

```diff
@@ -304,15 +299,15 @@ void expand_env_esc(char_u *srcp, char_u *dst, int dstlen, bool esc, bool one,
           mustfree = true;
         }
 #else
-        /* cannot expand user's home directory, so don't try */
+        // cannot expand user's home directory, so don't try
         var = NULL;
-        tail = (char_u *)"";            /* for gcc */
-#endif /* UNIX */
+        tail = (char_u *)"";  // for gcc
```

Comment:
I'm not sure that the for GCC comment is needed, as both it and Clang warn if the cast isn't there. Unless the comment isn't related to the cast...